src/List/List.js

```diff
@@ -45,13 +48,16 @@ export default class List extends Component {
       className: classNameProp,
       component,
       padding,
+      children: childrenProp,
       ...other,
     } = this.props;
     const classes = this.context.styleManager.render(styleSheet);
+    const children = Children.toArray(childrenProp);
```

Comment:
We could make this an explicit option instead of implicit behaviour when Subheader is the first child.